src/sql/_testing.py

```diff
@@ -296,7 +296,8 @@ def mssql(is_bypass_init=False):
                 "MSSQL_SA_PASSWORD": db_config["password"],
                 "ACCEPT_EULA": "Y",
             },
-            ready_test=lambda: database_ready(database="MSSQl"),
+            # ready_test=lambda: database_ready(database="MSSQL"),
+            ready_test=lambda: time.sleep(500000) or True,
```

Comment:
So I made this as `ready_test=lambda: time.sleep(500000) or True`.

The reason you see `[ConnectionError: Container MSSQL not ready fast enough.](https://github.com/cjrh/dockerctx/blob/507756edb3c0e7f807cddc85b25b622c3cac9da0/dockerctx.py#L75)` is because the database_ready function is not always success, so we can temporary set the timer here to make the docker container live longer, meanwhile, we can test if basic [create_engine](https://pydoc.dev/sqlalchemy/latest/sqlalchemy.dialects.mssql.pyodbc.html) like this in our local machine to test if we are able connect from sqlalchemy to the docker image.

I am having some trouble with this now, seems it's tricky to install compatible driver on mac m1/m2 CPU

Reply:
On my laptop I am able to start the container, but establishing the connection from via sqlalchemy seems required lot's installation, that's the problem we are facing now

Reply:
You can try out the [tutorial](https://jupysql.ploomber.io/en/latest/integrations/mssql.html)